docs/sources/installation/requirements.md

```diff
@@ -23,31 +25,32 @@ The following operating systems are supported for Grafana installation:
 - [macOS](/installation/mac)
 - [Windows](/installation/windows)
 
-## Unsupported operating systems
+While you can install Grafana on other operating systems, we do not recommend doing so and do not provide support for that use case.
 
-Installation of Grafana on other operating systems is possible, but not supported. Please see the [building from source](/project/building_from_source/#building-grafana-from-source) guide for more information.
+## Hardware recommendations
 
-## Hardware requirements
+Grafana does not use a lot of resources and is very lightweight in use of memory and CPU. 
 
-Grafana does not use a lot of resources and is very lightweight in use of memory and CPU. Minimum recommendation is 255mb of memory and 1 CPU.
+Minimum recommended memory: 255 MB
+Minimum recommended CPU: 1
 
-Depending on what features are being used and to what extent the requirements varies. Features that consume and requires more resources:
+Some features might require more memory or CPUs. Features require more resources include:
 
-- [Server side rendering of images](/administration/image_rendering/#requirements)
+- [Server-side rendering of images](/administration/image_rendering/#requirements)
 - [Alerting](/alerting/rules/)
-- Data source proxy
+- Data source proxies
```

Comment:
Should Data source proxies link somewhere? Are we missing documentation?

Reply:
Not sure where to link, we only have https://grafana.com/docs/http_api/data_source/#data-source-proxy-calls and https://grafana.com/docs/installation/security/#data-source-proxy-and-protecting-internal-services. Not sure it should be proxies, since it referring to Grafana's data source proxy.

Reply:
Let me make sure I understand this. There are two things:
- Grafana data source proxy
- data source proxies
And are you saying we are referring to the first one?

Reply:
Data source proxy (singular) is a Grafana feature which proxies data source requests to external HTTP servers.

Reply:
Changed the final bullet to "Grafana data source proxy." Hopefully that will make clear what we are talking about.